cdk/drupal/drupal_stack.py

```diff
@@ -1284,6 +1285,61 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
         )
         sg_https_ingress.cfn_options.condition = certificate_arn_exists_condition
 
+        # codebuild
+        codebuild_transform_service_role = aws_iam.Role(
+            self,
+            "CodeBuildTransformServiceRole",
+            assumed_by=aws_iam.ServicePrincipal('codebuild.amazonaws.com'),
+            inline_policies={
+                "TransformRolePermssions": aws_iam.PolicyDocument(
+                    statements=[
+                        aws_iam.PolicyStatement(
+                            effect=aws_iam.Effect.ALLOW,
+                            actions=[
+                                'logs:CreateLogGroup',
+                                'logs:CreateLogStream',
+                                'logs:PutLogEvents'
+                            ],
+                            resources=['*']
+                        ),
+                        aws_iam.PolicyStatement(
+                            effect=aws_iam.Effect.ALLOW,
+                            actions=[
+                                's3:GetObject',
+                                's3:PutObject'
+                            ],
+                            resources=[
+                                "arn:aws:s3:::github-user-and-bucket-taskcatbucket-2zppaw3wi3sx/*"
```

Comment:
should be a parameter.

Reply:
made this a conditional reference using Arn.format